buster/docparser.py

```diff
@@ -138,7 +101,7 @@ def read_documents(filepath: str) -> pd.DataFrame:
 
 def compute_n_tokens(df: pd.DataFrame) -> pd.DataFrame:
     encoding = tiktoken.get_encoding(EMBEDDING_ENCODING)
-    df["n_tokens"] = df.text.apply(lambda x: len(encoding.encode(x)))
+    df["n_tokens"] = df.text.apply(lambda x: len(encoding.encode(x, allowed_special={"<|endoftext|>"})))
```

Comment:
maybe add a comment here that we should look into what the downsides of allowing this sepcial character might be